.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - '10'
+  - '16'
```

Comment:
please also add an `.nvmrc` file that contains this version :)
GitHub Actions can then directly [read the configured version from there](https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file), same as users' `nvm` installation.
Not sure about travis.

Reply:
This will be part of https://github.com/bitmovin/bitmovin-player-ui/pull/500 :) I can add here too if you like though

Reply:
Added in 3a62a045